Deeploy/Targets/Spatz/Bindings.py

```
@@ -0,0 +1,97 @@
+    MemoryManagementGeneration(),
+])
+
+SpatzGatherBindings = [
+    NodeBinding(
+        GatherChecker(
+            [PointerClass(float32_t), PointerClass(type)],
+            [PointerClass(float32_t)]
+        ),
+        GatherTemplate.dynamicDMAtemplate,
+        TiledTransformer
+    ) for type in IntegerDataTypes
```

Ruff (0.15.20)

[error] Variable `type` is shadowing a Python builtin

(A001)